### PR TITLE
A whole new bunch of foods and drinks

### LINKED ISSUE #8376: Kebab not properly removed
When you create a custom kebab (maybe normal kebabs too?) the new kebab ends up on the floor, but there's still something in the hand that was holding the rod. It looks like the rod, but you can't use it for anything and all you can really do is drop it to clear it from existence.

### LINKED ISSUE #8482: Reagent volume too low for some foods.
Why is this an issue? Between crafted food inheriting reagents from ingredients and upgraded microwaves boosting nutriment and vitamin, some food items hit their volume cap and lose some of their reagents as a result.

For just one example, I made a meatbread made from cutlets cooked in a ultra-high power micro laser upgraded microwave, and it had...
40.5 Nutriment and 9.5 vitamin

While a meat bread made with roundstart microwaves contained...
33 Nutriment and 16 vitamin.

In my opinion, sacrificing all that vitamin for nutriment is a downgrade.

### LINKED ISSUE #8717: Storing items in food is broken.
You can no longer put items in cheesewheels or loafs of bread, though it still works for cakes.

EDIT: More specifically it works with at least cheesecakes, I haven't tried the others.

### PR DESCRIPTION
**Pull request by phil235**
- Splitting food.dmi into smaller files.
- Repathing meat slab, meatsteaks and cutlets into meat/slab, meat/steak, meat/rawcutlet, meat/cutlet.
- Fixes issue with food naming when containing meat stuff, especially human meat. (a consequence of this is no longer being able to use special breadslice/cakeslice as first ingredient in bread/cake to avoid infinite bread-ception and inifinitely long names).
- Bowl filled with reagent now gets a nice half transparent overlay with the reagents colors (like beaker)
- Mousedroping a snack that's on a table now opens the tablecrafting window.
- Fixes not being to store items in plain bread, plain cake and cheesewheel. Fixes #8717 
- Changes the glasses of what? to use an overlay with reagents color.
- Adding a ricepack to the kitchen cabinet and the food crate.
- Custom food filling color is now a mix instead of the color of the last ingredient.
- Adding more bowl to the dinnerware vending machine.
- Adding pumpkinjuice, blumpkingjuice, cherryshake, bluecherryshake pumpkin latte, drunken blumpkin reagents.
- Fixes bear migraw recipe eating the lighter.
- Fixes not being able to break eggs in drinking glasses.
- Fixes #8376 
- Fixes #8482 
- Fixes human slab/cutlet/steak not giving their name to custom food.

Sprites by me, chocobro and some borrowed from bay.
Recipe ideas by me, chocobro and donkhonk.

### CODE REVIEW

**Comment by RandomMarine** [comment]
I think #8482 is still in effect for super bite burgers.

File: code/modules/food&drinks/food/snacks_pizza.dm [hunk 1]
```diff
@@ -23,51 +25,89 @@
...
+	list_reagents = list("nutriment" = 25, "doctorsdelight" = 5, "tomatojuice" = 6, "vitamin" = 5)
+
+/obj/item/weapon/reagent_containers/food/snacks/pizzaslice/dank
+	name = "dank pizza slice"
+	desc = "So good, man..."
+	icon_state = "dankpizzaslice"
+	filling_color = "#2E8B57"
+
+/obj/item/weapon/reagent_containers/food/snacks/pizza/sassysage
+	name = "vegetable pizza"
```
Comment (Incoming5643) [inline code/modules/food&drinks/food/snacks_pizza.dm]: This and the dank pizza name on line 85 seem to have not been renamed properly.

**Comment by Incoming5643** [comment]
Remember that adding new food items is pretty pointless unless you or someone else manages to update the wiki afterwards

File: code/modules/food&drinks/food/snacks_pizza.dm (hunk 1 above)
Comment (phil235) [inline code/modules/food&drinks/food/snacks_pizza.dm]: Indeed, I'll fix it.

**Comment by phil235** [comment]
I'll update the wiki myself, just like I did for the new cooking system. But now that tablecrafting shows you the recipes that include some of the items on the table we don't need the wiki as much as before.

**Comment by Cheridan** [comment]
That sprite at the top right is horrible (nachos?), obviously a resized photo.

**Comment by phil235** [comment]
That's roast parsnip, it's chocobro's sprite. I can try to resprite it if you want.

**Comment by Cheridan** [comment]
That'd be great, the hi-def sprite *really* clashes with the rest of the game

**Comment by Chocobro** [comment]
tfw i made parsnip sprite and took like four hours on it then it's not used
(i actually fully hated it)

**Pull request merged**